packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -450,6 +437,66 @@ export default class VTable {
         }
         return result;
     }
+
+    private setEmptyTableCells() {
+        for (let i = 0, row; (row = this.table.rows[i]); i++) {
+            for (let j = 0, cell; (cell = row.cells[j]); j++) {
+                if (cell) {
+                    if (!cell.innerHTML || !cell.innerHTML.trim()) {
+                        cell.appendChild(document.createElement('br'));
+                    }
+                }
+            }
+        }
+    }
+
+    public setTableCells() {
+        // measure and store the width/height into VCell
+        for (let i = 0; i < this.cells.length; i++) {
+            for (let j = 0; j < this.cells[i].length; j++) {
+                const cell = this.cells[i][j];
+                if (cell.td) {
+                    const rect = cell.td.getBoundingClientRect();
+                    cell.width = rect.width;
+                    cell.height = rect.height;
+                }
+            }
+        }

```

Comment:
This part can be moved into constructor of VTable since it just read the size and do no change to the table. So that even normalizeSize is false, we can use the width/height value of VCell.

Reply:
I think we should do this logic only when we need to resize the table, we don't need to set/use width/height of VCell when we are not resizing the table.

Reply:
For table resize, yes it is only used when we need to do resize. But as a public interface, other people may also want to use it in other cases. So we should make sure it always have correct value. Otherwise, need to comment it to explain when the value will be available.

RoosterJs is open source project, each line of public code is not only for ourselves, but also for people who may use it. Keep that in mind.

Reply:
discussed offline and fixed